features/admin.roles.v2/components/edit-role/edit-role-users.tsx

```diff
@@ -410,7 +410,10 @@ export const RoleUsersList: FunctionComponent<RoleUsersPropsInterface> = (
                             {
                                 users && availableUserStores && !isReadOnly && (
                                     <Grid container spacing={ 1 }>
-                                        { !activeUserStore && (
+                                        { (
+                                            !activeUserStore
+                                            || activeUserStore === userstoresConfig?.primaryUserstoreName
```

Comment:
The ideal fix is to improve the logic in https://github.com/wso2/identity-apps/blob/master/features/admin.console-settings.v1/components/console-roles/console-roles-edit/console-roles-edit.tsx#L221

Reply:
What is the ideal fix?

Reply:
activeUserStore prop is used to switch between the primary userstore and the shared userstore in the saas deployment, which is only available when privileged user feature is available. So, updating the condition in https://github.com/wso2/identity-apps/blob/master/features/admin.console-settings.v1/components/console-roles/console-roles-edit/console-roles-edit.tsx#L221 to include ‎`activeUserStore === userstoresConfig?.primaryUserstoreName`should show the userstore dropdown when `<orgname> organization` radio button is selected.

Reply:
This is introduced with https://github.com/wso2/identity-apps/pull/7050. I will check if we can improve with the above fix.